.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push Docker image
+        env:
+          DOCKER_USERNAME: argoscs
+          IMAGE_NAME: argoscs/ui
+        run: |
+          echo "Building Docker image..."
+          docker build -t ${IMAGE_NAME} -f Dockerfile.dev .
```

Comment:
here you should use the dockerfile for production, not the development one

Reply:
Solved